### PR TITLE
Add Windows support for getProcesses

### PR DESCRIPTION
**Pull request by davidcralph**
This uses a very nice module called ps-list.

I actually tested this PR and it works fine for me on Windows 10.

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -228,15 +229,15 @@ module.exports = {
...
-   * @works Linux, MacOS
+   * @works Windows, Linux, MacOS
    * @returns An array of all the processes running by the `pid` argument
    */
-  getProcesses(pid = 10) {
-    if (this.getPlatform() === 'Windows') throw new SyntaxError('The "getProcesses" function requires to be in a Unix-subsystem');
+  async getProcesses(pid = 10) {
+    if (this.getPlatform() === 'Windows') {
+      const processes = await psList();
+      return processes.slice(0, 10);
```
Comment (auguwu) [inline lib/index.js]: Should it slice how many the `pid` argument should give?

File: index.d.ts [hunk 2]
```diff
@@ -155,11 +155,8 @@ declare module '@augu/sysinfo' {
...
    * 
-   * Warnings:
-   * - This will not work in a Windows machine
-   * 
    * @param pid The number of processes to list
-   * @works Linux, MacOS
+   * @works Windows, Linux, MacOS
    * @returns An array of all the processes running by the `pid` argument
    */
   export function getProcesses(pid?: number): Process[];
```
Comment (auguwu) [inline index.d.ts:162]: You sure it returns the `Process` interface when on Windows?

**Review by auguwu — COMMENTED** [review]
This PR is good, thanks.
I'll merge when you do my review.

File: lib/index.js (hunk 1 above)
Comment (davidcralph) [reply lib/index.js]: Forgot about this

File: index.d.ts (hunk 2 above)
Comment (davidcralph) [reply index.d.ts:162]: no idea my knowledge of typescript is smol

Comment (auguwu) [reply index.d.ts:162]: I'll do a fix for this I guess

**Pull request merged**